src/player-config.js

```diff
@@ -24,6 +24,7 @@ const DefaultConfig = {
     loop: false,
     allowMutedAutoPlay: true,
     muted: false,
+    playbackRate: [0.5, 1, 2, 4],
```

Comment:
This is not correct, the config needs to be taken from the engine as only it knows what he can support.
Adding a default will cause the player to return this instead of engine supported playback rates

Reply:
what's about this?